### PR TITLE
Updated title

### PR DESCRIPTION
**Pull request by h4v0kr**

### CODE REVIEW

File: How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md [hunk 1]
```diff
@@ -36,20 +36,20 @@ rpcuser=yourusername
...
 
-Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
+Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs e.g. bitcoind. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
 
 ![](images/gmtcZSJ.png)
 
-To restart your container after an OS update or restart, select the `Enable auto-restart`.
-From the  `Volume`, select `Add Folder`, enter your local path in `File/Folder` and `/bitcoin/.bitcoin` as `Mount path`.
+To restart your container after an OS update or reboot, select the `Enable auto-restart`.
+From the `Volume` tab, select `Add Folder`, enter your local path in `File/Folder` and `/bitcoin/.bitcoin` as `Mount path`.
```
Comment (h4v0kr) [inline How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md]: Are these correct as per the instruction @roshii ?  The tutorial suggests were setting up a bitcoin container in testnet mode. Should we update this screen to reflect the aforementioned bitcoin-testnet folder instead?

File: How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md [hunk 2]
```diff
@@ -36,20 +36,20 @@ rpcuser=yourusername
...
+ We are now ready to install our bitcoin docker container by opening the `Docker` package, then selecting `Registry` and searching for "bitcoind". I recommend downloading the `kylemanna` image which will simply install bitcoind on an Ubuntu instance from its official repository - feel free to inspect the code at [Github](https://github.com/kylemanna/docker-bitcoind/blob/master/Dockerfile)
 
 ![](images/wiZtXhR.png)
 
-Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
+Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs e.g. bitcoind. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
```
Comment (h4v0kr) [inline How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md]: Perhaps we should update this screen to show `bitcoind-testnet`?

File: How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md [hunk 3]
```diff
@@ -36,20 +36,20 @@ rpcuser=yourusername
 rpcpassword=yourpassword
```
Comment (h4v0kr) [inline How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md]: Any way we can remove `decred` from this screenshot?

File: How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md (hunk 1 above)
Comment (roshii) [reply How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md]: this is correct yes, local path = `bitcoin-testnet` bit I should make this more clear

**Comment by roshii** [comment]
I'll push changes tomorrow

**Comment by roshii** [comment]
Everything is pushed ;)

**Pull request merged**